mobile/lib/models/recording_info.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:audio_service/audio_service.dart';
+
+class RecordingInfo {
+  RecordingInfo.fromJson(Map<String, dynamic> jsonData)
+      : title = jsonData['acf']['title'] as String,
+        thumbnailPath = (jsonData['acf']['image'] as int).toString(),
+        recordingPath = (jsonData['acf']['file'] as int).toString(),
+        duration = Duration.zero;
+
+  final String title;
+  String thumbnailPath;
+  String recordingPath;
+  Duration duration;
+
+  MediaItem get mediaItem => MediaItem(
+        id: recordingPath,
+        title: title,
+        artist: 'Radio Aktywne',
```

Comment:
At some point we might want to consider using `localize` here... maybe add a TODO?